Library/Homebrew/attestation.rb

```diff
@@ -62,6 +62,9 @@ def self.enabled?
       return false if ENV.fetch("CI", false)
       return false if OS.unsupported_configuration?
 
+      gh_version = Formula["gh"].any_installed_version.to_s
+      return false if gh_version.nil? || gh_version < "2.49"
```

Comment:
Per https://github.com/Homebrew/brew/pull/18030#discussion_r1714297330, I think this should be:

```suggestion
      gh_version = Formula["gh"].any_installed_version
      return false if gh_version.nil? || gh_version.version < "2.49"
```

(The current version will crash if `any_installed_version.nil?`, and might be wrong if version counting diverges from lexicographic order 🙂)